torcharrow/velox_rt/string_column_cpu.py

```diff
@@ -287,6 +287,23 @@ def contains_re(
             self._parent.dtype.nullable
         )
 
+    def extract_all_re(
```

Comment:
It's `findall_re` and there is no capture group in `findall_re`:

https://github.com/facebookresearch/torcharrow/blob/6ba606b34bc1cd7801745519a396742056a96c2d/torcharrow/istring_column.py#L202-L211